scripts/job_management.py

```diff
@@ -438,6 +484,9 @@ def main():
                 args.restrict_to_samples_wildcards.append("*")
             for wildcard in args.restrict_to_samples_wildcards:
                 input_ntuples_list += fnmatch.filter(all_files,wildcard)
+        input_ntuples_list = [ e for e in  input_ntuples_list if "SUSY" not in e]
```

Comment:
Why do you remove SUSY? Friend trees for SUSY samples are needed for SVFit...

You could do something like in the lines above, just the other way round, so rejecting some wildecards. Otherwise please leave this line out.

Reply:
ups, sorry this commit was not meant to be in there